bin/rmt-cli

```diff
@@ -1,29 +1,27 @@
 #!/usr/bin/env ruby
 
-require 'etc'
-
 rmt_path = File.expand_path('..', __dir__)
 require_relative '../config/boot'
 $LOAD_PATH.unshift File.join(rmt_path, 'lib')
 
+require 'etc'
 require 'active_support'
 require 'active_record'
 require 'erb'
 require 'yaml'
 require 'rmt/config'
 
-# don't run as root
-if Process.euid == 0
+relative_load_paths = %w[lib lib/rmt app/models app/services].map { |dir| File.join(rmt_path, dir) }
+ActiveSupport::Dependencies.autoload_paths += relative_load_paths
+
+if RMT::CLI::Base.process_user_name == 'root'
```

Comment:
I don't see why this change would be required.

Reply:
This change is not required, but with `process_user_name` it is now possible to write descriptive code without needing a comment, which I learned is the preferred way,